src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -417,24 +419,46 @@ public async Task<JObject> Resolve(InvocationExpressionSyntax method, Dictionary
                     using var commandParamsObjWriter = new MonoBinaryWriter();
                     if (isTryingLinq == 0)
                         commandParamsObjWriter.WriteObj(objectId, context.SdbAgent);
+
                     if (method.ArgumentList != null)
                     {
-                        commandParamsObjWriter.Write((int)method.ArgumentList.Arguments.Count + isTryingLinq);
+                        var methodInfo = await context.SdbAgent.GetMethodInfo(methodId, token);
+                        int passedParamCnt = method.ArgumentList.Arguments.Count;
+                        var methodParamsInfo = new List<ParameterInfo>();
+                        if (methodInfo?.Info != null)
```

Comment:
When can this be null?

Reply:
E.g calling .ToList() on an object. So each non-user function.